spec/controllers/api/v1/workflows_controller_spec.rb

```diff
@@ -508,6 +508,21 @@
     end
   end
 
+  describe '#retired_subjects' do
+    before do
+      default_request scopes: scopes, user_id: authorized_user.id
+    end
+
+    let(:subject_set_project) { project }
+    let(:subject_set) { create(:subject_set, project: project, workflows: [project.workflows.first]) }
+    let(:subject_set_id) { subject_set.id }
+    let(:subject) { create(:subject, subject_sets: [subject_set]) }
+
+    it 'retires the subject' do
+      post :retired_subjects, workflow_id: workflow.id, subject_id: subject.id
```

Comment:
The route works fine in dev server, but tests cant find it. @camallen @parrish any ideas?